<?xml version='1.1' encoding='UTF-8'?>
<Tag plugin="workflow-support@3.4">
  <node class="cps.n.StepAtomNode" plugin="workflow-cps@2.80">
    <parentIds>
      <string>17</string>
    </parentIds>
    <id>18</id>
    <descriptorId>org.jenkinsci.plugins.workflow.support.steps.input.InputStep</descriptorId>
  </node>
  <actions>
    <cps.a.ArgumentsActionImpl plugin="workflow-cps@2.80">
      <arguments>
        <entry>
          <string>message</string>
          <string>User input required</string>
        </entry>
        <entry>
          <string>parameters</string>
          <list>
            <org.jenkinsci.plugins.structs.describable.UninstantiatedDescribable plugin="structs@1.20">
              <symbol>choice</symbol>
              <arguments class="linked-hash-map">
                <entry>
                  <string>name</string>
                  <string>RELEASE_SCOPE</string>
                </entry>
                <entry>
                  <string>choices</string>
                  <string>Patch
Minor
Major</string>
                </entry>
                <entry>
                  <string>description</string>
                  <string>What is the release scope?</string>
                </entry>
              </arguments>
            </org.jenkinsci.plugins.structs.describable.UninstantiatedDescribable>
          </list>
        </entry>
      </arguments>
      <isUnmodifiedBySanitization>true</isUnmodifiedBySanitization>
    </cps.a.ArgumentsActionImpl>
    <wf.a.TimingAction plugin="workflow-api@2.40">
      <startTime>1594754851072</startTime>
    </wf.a.TimingAction>
    <s.a.LogStorageAction/>
    <s.a.PauseAction>
      <cause>Input</cause>
      <startTime>1594754851086</startTime>
      <endTime>1594754896039</endTime>
    </s.a.PauseAction>
    <wf.a.ErrorAction plugin="workflow-api@2.40">
      <error class="org.jenkinsci.plugins.workflow.steps.FlowInterruptedException" plugin="workflow-step-api@2.22">
        <stackTrace>
          <trace>org.jenkinsci.plugins.workflow.support.steps.input.InputStepExecution.doAbort(InputStepExecution.java:231)</trace>
          <trace>org.jenkinsci.plugins.workflow.support.steps.input.InputStepExecution$1.run(InputStepExecution.java:104)</trace>
          <trace>jenkins.security.ImpersonatingScheduledExecutorService$1.run(ImpersonatingScheduledExecutorService.java:58)</trace>
          <trace>java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)</trace>
          <trace>java.util.concurrent.FutureTask.run(FutureTask.java:266)</trace>
          <trace>java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180)</trace>
          <trace>java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293)</trace>
          <trace>java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)</trace>
          <trace>java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)</trace>
          <trace>java.lang.Thread.run(Thread.java:748)</trace>
        </stackTrace>
        <suppressedExceptions class="java.util.Collections$UnmodifiableRandomAccessList" resolves-to="java.util.Collections$UnmodifiableList">
          <c class="list"/>
          <list reference="../c"/>
        </suppressedExceptions>
        <result>
          <name>ABORTED</name>
          <ordinal>4</ordinal>
          <color>ABORTED</color>
          <completeBuild>false</completeBuild>
        </result>
        <causes class="java.util.Arrays$ArrayList">
          <a class="jenkins.model.CauseOfInterruption-array">
            <org.jenkinsci.plugins.workflow.support.steps.input.Rejection plugin="pipeline-input-step@2.11">
              <userName>SYSTEM</userName>
              <timestamp>1594754896030</timestamp>
            </org.jenkinsci.plugins.workflow.support.steps.input.Rejection>
          </a>
        </causes>
        <actualInterruption>true</actualInterruption>
      </error>
    </wf.a.ErrorAction>
  </actions>
</Tag>